deepl-connector-demo/src_hd/com/axonivy/connector/deepl/demo/translate/translate.xhtml

```diff
@@ -0,0 +1,46 @@
+<html xmlns="http://www.w3.org/1999/xhtml"
+	xmlns:f="http://xmlns.jcp.org/jsf/core"
+	xmlns:h="http://xmlns.jcp.org/jsf/html"
+	xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
+	xmlns:ic="http://ivyteam.ch/jsf/component"
+	xmlns:p="http://primefaces.org/ui"
+	xmlns:pe="http://primefaces.org/ui/extensions">
+<h:body>
+	<ui:composition template="/layouts/frame-10.xhtml">
+		<ui:define name="title">translate</ui:define>
+		<ui:define name="content">
+
+			<h:form id="form">
+
+				<div style="display: flex; align-items: flex-end;">
+					<div style="display: flex; flex-direction: column;">
+						<p:outputLabel for="inputText" value="inputText" />
+						<p:inputTextarea rows="6" cols="33" id="inputText"
+							value="#{data.data.inputText}" />
+					</div>
+					<div style="display: flex; flex-direction: column;">
+						<div>
+							<p:selectOneMenu id="language" value="#{data.translate}"
+								converter="ivy.ListItem">
+								<f:selectItems value="#{data.languages}" var="language"
+									itemValue="#{language}" itemLabel="#{language.getValue()}">
+								</f:selectItems>
+							</p:selectOneMenu>
+							<p:outputLabel for="outputText" value="outputText" />
+						</div>
+
+						<p:inputTextarea rows="6" cols="33" id="outputText"
+							value="#{data.data.OutputText}" />
+					</div>
+				</div>
+				<div class="command-btns" style="justify-content: left;">
+					<p:commandButton id="translate" actionListener="#{logic.translate}"
+						value="translate" update="form" icon="si si-chat-translate" />
+				</div>
+			</h:form>
+
+		</ui:define>
+	</ui:composition>
+</h:body>
```

Comment:
let's show this dialog to @andreasbalsiger and ask for input in the next break :dango: